sqlmesh/core/engine_adapter/mssql.py

```diff
@@ -40,25 +42,53 @@ class MSSQLEngineAdapter(
 
     DIALECT: str = "tsql"
     FALSE_PREDICATE = exp.condition("1=2")
```

Comment:
I believe @tobymao said this is no longer needed and you can just put `exp.false()`. I believe SQLGlot now converts that to something like `1=2`. If that is the case then this `FALSE_PREDICATE` can be completely removed since it was just added for mssql.